mutagen/wavpack.py

```diff
@@ -90,10 +91,12 @@ def __init__(self, fileobj):
         self.version = header.version
         self.channels = bool(header.flags & 4) or 2
         self.sample_rate = RATES[(header.flags >> 23) & 0xF]
+        self.bits_per_sample = ((header.flags & 3) + 1) * 8
 
         # most common multiplier (DSD64)
         if (header.flags >> 31) & 1:
             self.sample_rate *= 4
+            self.bits_per_sample = 1
```

Comment:
This isn't needed now I think

Reply:
> This isn't needed now I think

This is needed since DSD uses 1 bit, but if you don't use it bits_per_sample is 8 because we have `1 * 8`. `wvunpack -qs` does the same (https://github.com/dbry/WavPack/blob/master/cli/wvunpack.c#L2200)

Reply:
ah, good point